### PR TITLE
Update CI Script to run UI test

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -42,6 +44,22 @@ warn() {
...
 }
 
+install-node() {
+  info Installing node
+
+  curl -sL https://deb.nodesource.com/setup_14.x | bash -
+  apt-get install -y nodejs
+
+  npm install
+
```
Comment (sthaha) [inline test/presubmit-tests.sh:54]: could you move `npm install` outside this function into `build`? 
Also, could you please run `node version` ?

File: test/presubmit-tests.sh [hunk 2]
```diff
@@ -77,6 +95,13 @@ api-build(){
   go build -mod=vendor ./cmd/...
 }
 
+ui-build(){
+  cd $UI_DIR
+  install-node
+
+  CI=true npm run build
```
Comment (sthaha) [inline test/presubmit-tests.sh:102]: why do we need to export CI=true?

File: test/presubmit-tests.sh [hunk 3]
```diff
@@ -77,6 +95,13 @@ api-build(){
   go build -mod=vendor ./cmd/...
 }
 
+ui-build(){
+  cd $UI_DIR
+  install-node
```
Comment (sthaha) [inline test/presubmit-tests.sh:100]: could you run `npm install` here ?

**Comment by sthaha** [comment]
@PuneetPunamiya looks good overall, some minor comments to fix 👼

**Review by vdemeester — COMMENTED** [review]
/lgtm

**Review by sthaha — COMMENTED** [review]
/lgtm
/approve
/meow

**Pull request merged**